x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -87,6 +88,11 @@ const errorHandler = <E extends Error>(
     return res.badRequest({ body: { message: error.message, attributes: { type: error.type } } });
   }
 
+  if (error instanceof TrustedAppPolicyPermissionsError) {
+    logger.error(error);
+    return res.badRequest({ body: { message: error.message, attributes: { type: error.type } } });
```

Comment:
Suggestion: just send the `error`:

```suggestion
    return res.badRequest({ body: error });
```